runtimes/peregrine/src/lib.rs

```diff
@@ -1110,6 +1123,7 @@ mod benches {
 		[pallet_vesting, Vesting]
 		[pallet_proxy, Proxy]
 		[pallet_xcm, PolkadotXcm]
+		[pallet_multisig, Multisig]
```

Comment:
Maybe we can put this also in the same position as the pallet is added to the runtime?

Reply:
In the `frame_benchmarking::define_benchmarks!` macro, the items are separated by KILT and FRAME pallets. Should I rearrange the whole macro then?

Reply:
I would suggest to keep the current order. Maybe we can fix the comments and the order in a separate PR?

Reply:
I created a [ticket] (https://github.com/KILTprotocol/ticket/issues/2621)